src/harness/runnerServer.ts

```diff
@@ -0,0 +1,17 @@
+﻿

```

Comment:
Whoops! Yes, this was me prototyping for a different improvement and then forking off (pun intended) to work on this.